ci/kind/test-upgrade-antrea.sh

```diff
@@ -152,10 +152,10 @@ function version_lt() { test "$(printf '%s\n' "$@" | sort -rV | head -n 1)" != "
 
 DOCKER_IMAGES=()
 if version_lt "$FROM_TAG" v1.15; then
-    DOCKER_IMAGES+=("projects.registry.vmware.com/antrea/antrea-ubuntu:$FROM_TAG")
+    DOCKER_IMAGES+=("antrea/antrea-ubuntu:$FROM_TAG")
```

Comment:
Should we keep this as before considering it's only for testing? I am thinking the test may fail more frequently due to Docker rate limit.

Reply:
Another question is should we release a new patch release after the backport? cc @tnqn

Reply:
We don't have rate-limit as explained in the PR description.
Yes, we should release new patch releases, there has been some bugfixes accumulated.

Reply:
I am not getting that why we change the images from Dockerhub to Harbor registry at the first place? I recalled it was because of rate limits? Or Antrea wasn't a  part of the "Sponsored OSS" programs when the registry is changed?

Reply:
Right